### PR TITLE
Preparing release 5.2.0

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SERVER_CONFIG_SAMPLE = config/example.ini
 SERVER_CONFIG_LOCAL = config/development.ini
-VIRTUALENV = virtualenv --python python3.5
+VIRTUALENV = virtualenv --python python3.6
```
Comment (Natim) [inline Makefile]: Should we upgrade the Dockerfile Python version too?

File: requirements.txt [hunk 2]
```diff
@@ -74,7 +74,7 @@ Unidecode==1.0.22
 urllib3==1.22
 venusian==1.1.0
 waitress==1.1.0
-WebOb==1.7.4
+WebOb==1.8.0rc1
```
Comment (Natim) [inline requirements.txt]: Avoid using the RC here.

**Comment by leplatrem** [comment]
Failing because of this, @Natim seen this before?
```
+kinto-wizard load tests/amo-blocklist.yaml --server http://localhost:8888/v1 --auth=user:pass --bucket staging
Traceback (most recent call last):
  File "/home/travis/gopath/src/github.com/mozilla-services/kinto-dist/.venv/bin/kinto-wizard", line 11, in <module>
    sys.exit(main())
  File "/home/travis/gopath/src/github.com/mozilla-services/kinto-dist/.venv/lib/python3.6/site-packages/kinto_wizard/__main__.py", line 90, in main
    force=args.force
  File "/usr/lib/python3.6/asyncio/base_events.py", line 467, in run_until_complete
    return future.result()
  File "/home/travis/gopath/src/github.com/mozilla-services/kinto-dist/.venv/lib/python3.6/site-packages/kinto_wizard/yaml2kinto.py", line 62, in initialize_server
    permissions=bucket_permissions)
  File "/home/travis/gopath/src/github.com/mozilla-services/kinto-dist/.venv/lib/python3.6/site-packages/kinto_http/__init__.py", line 311, in patch_bucket
    if_match=if_match,
  File "/home/travis/gopath/src/github.com/mozilla-services/kinto-dist/.venv/lib/python3.6/site-packages/kinto_http/__init__.py", line 193, in _patch_method
    headers=headers)
TypeError: request() got an unexpected keyword argument 'payload'
````

**Comment by leplatrem** [comment]
Oh it looks like you were right about the breaking change in kinto-http then.

**Comment by leplatrem** [comment]
> Oh it looks like you were right about the breaking change in kinto-http then.

It's a bug in kinto-http : `BatchSession.request()` missing payload.

**Comment by leplatrem** [comment]
On hold because of https://github.com/mozilla-services/kinto-amo/pull/26/files

And discussions in https://bugzilla.mozilla.org/show_bug.cgi?id=1391792

File: setup.py [hunk 3]
```diff
@@ -43,6 +43,7 @@ def read_file(filename):
           "Programming Language :: Python :: 3",
           "Programming Language :: Python :: 3.4",
```
Comment (Natim) [inline setup.py]: Shall we remove this too :)

**Pull request merged**